src/cpu/aarch64/jit_sve_512_x8s8s32x_conv_kernel.hpp

```diff
@@ -0,0 +1,296 @@
+/*******************************************************************************
+* Copyright 2016-2020 Intel Corporation
```

Comment:
[here and other places]
nit:  Since it's a new file so just the year `2020` [or maybe `2021` depending upon when it is promoted] is enough.

Reply:
Thx! I'll change it.